package.json

```diff
@@ -14,7 +14,13 @@
     "lint": "eslint . --ignore-path .gitignore",
     "test:unit": "vitest",
     "test:coverage": "vitest --coverage",
-    "test:e2e": "playwright test"
+    "test:e2e": "playwright test",
+    "test:e2e:1": "playwright test --shard=1/6",
```

Comment:
Do you actually need these? You can pass command line args through to the npm scripts, so couldn't you just call
`npm run test:e2e -- --shard=${{ inputs.shard }}/6` from the action?

Reply:
thanks, that is much better! also done it in one workflow file!